tests/ServerSpec.hs

```diff
@@ -246,7 +237,7 @@ serverSpec = do
         -- First send an invalid webhook with a bad payload.
         let badPayload = "this is definitely not valid json"
         badResponse <- httpPostGithubEvent "/hook/github" "status" badPayload
-        Http.getResponseStatus badResponse `shouldBe` internalServerError500
+        Http.getResponseStatus badResponse `shouldBe` badRequest400
```

Comment:
Is this caused by a version upgrade of one of our dependencies?

Reply:
Probably `scotty` at least I did not change anything related. It's what happens on unparseable (or undecodable) JSON